ee/modules/030-cloud-provider-vsphere/templates/csi-legacy/controller.yaml

```diff
@@ -55,7 +55,12 @@
 {{- $_ := set $csiNodeConfig "nodeImage" $csiControllerImage }}
 {{- $_ := set $csiNodeConfig "driverFQDN" "vsphere.csi.vmware.com" }}
 {{- $_ := set $csiNodeConfig "additionalNodeEnvs" (include "csi_node_legacy_envs" . | fromYamlArray) }}
+{{- $_ := set $csiNodeConfig "additionalNodeEnvs" (include "csi_node_envs" . | fromYamlArray) }}
```

Comment:
`csi_node_legacy_envs` will be overwritten?

Reply:
https://github.com/deckhouse/deckhouse/pull/10676/commits/7ed2571d74226cff326d804febed92e28c7a3c3c

Reply:
https://github.com/deckhouse/deckhouse/pull/10676/commits/d76a639329c29593bf2328408f398e6b6eafea07